hibernate/cache-commons/src/test/java/org/infinispan/test/hibernate/cache/commons/functional/cluster/EntityCollectionInvalidationTest.java

```diff
@@ -144,6 +150,14 @@ public void testAll() throws Exception {
 		// and therefore the putFromLoad in getCustomer could be considered stale if executed too soon.
 		timeService.advance(1);
 
+      CountDownLatch remoteCollectionLoadLatch = null;
+      if (!cacheMode.isInvalidation() && (mode.equals("nonstrict") || mode.equals("read-write"))) {
```

Comment:
shouldn't a non-invalidation mode imply either nonstrict or read-write?

Reply:
Read only?

Reply:
That's not tested in this test (see `getParameters`). Besides, putFromLoads happen in RO exactly the same way.